src/components/CodeView/utils.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { RefractorNode } from 'refractor';
+import makeClassName from 'classnames';
+
+const mapChild = (
+  child: RefractorNode,
+  i: number,
+  depth: number,
+): React.ReactElement | string => {
+  if (child.tagName) {
+    // @ts-ignore
```

Comment:
It looks like `properties` was duplicated, right? This seems to fix it:

```diff
diff --git a/src/@types/refractor/index.d.ts b/src/@types/refractor/index.d.ts
index b6a6f3f..f5b809e 100644
--- a/src/@types/refractor/index.d.ts
+++ b/src/@types/refractor/index.d.ts
@@ -3,7 +3,6 @@ declare module 'refractor' {
 
   type RefractorNode = {
     children: RefractorNode[];
-    properties: object;
     tagName: string;
     type: string;
     value: string;
diff --git a/src/components/CodeView/utils.tsx b/src/components/CodeView/utils.tsx
index 546e1ae..64c8f4b 100644
--- a/src/components/CodeView/utils.tsx
+++ b/src/components/CodeView/utils.tsx
@@ -8,8 +8,9 @@ const mapChild = (
   depth: number,
 ): React.ReactElement | string => {
   if (child.tagName) {
-    // @ts-ignore
-    const className = makeClassName(child.properties.className);
+    const className = makeClassName(
+      (child.properties && child.properties.className) || '',
+    );
 
     return React.createElement(
       child.tagName,
```

Reply:
oops, thanks

Reply:
OMG, I actually never realized `properties` was duplicated. I am so stupid... I just realized now while trying to fix the issue with your patch 😥